Sources/FuzzilliCli/Profiles/QtjsProfile.swift

```diff
@@ -0,0 +1,114 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+import Fuzzilli
+fileprivate let ForceQV4JITGenerator = CodeGenerator("ForceQV4JITGenerator", input: .function()) { b, f in 
```

Comment:
What is QV4? Does QTJS have a JIT engine?

Reply:
So QV4 is the execution engine which is a fork of V8 and contains its own JIT engine. 

The Qt QJSEngine is a wrapper for the engine which exposes a .evaluate() function that is being called by the instrumentation. QJSEngine.evaluate() handles parsing and creation of a scope for the script as well as error catching and subsequent reporting via a parameterized pointer to a list of exception strings. 

I understand if naming might be confusing, so if you have a better idea for the naming of the target I can change it ASAP

Reply:
I see, thanks for the explanation! Maybe just add a quick comment here that `// QV4 is the execution engine behind QTJS` or so?

Reply:
you got it!